src/util/util.ts

```diff
@@ -37,3 +37,20 @@ export const isSubstringOf = (substring: string, source: string): boolean =>
   pipe(source, S.toLowerCase, S.includes(S.toLowerCase(substring)))
 
 export const isError = (value: unknown): value is Error => value instanceof Error
+
+export const escapeSparqlStr = (str?: string): string => {
+  const sparqlEscapeRegExp = /[\t\n\r\b\f"'\\]/g
```

Comment:
I think you can avoid the use of the regexp here, if you have the mapping of escaped characters. It will be more efficient, and more maintenable, imho.

Reply:
The function has been refactored according to you feedbacks in [05745c0](https://github.com/okp4/dataverse-portal/pull/371/commits/05745c05fd013b3864ec2d203d0c22c4ef3792e1). Thanks